_i18n/template/resources/developer-guides/wallet-rpc.md

```diff
@@ -168,15 +212,19 @@ Inputs:
 * *priority* - unsigned int; Set a priority for the transaction. Accepted Values are: 0-3 for: default, unimportant, normal, elevated, priority.
 * *do_not_relay* - boolean; (Optional) If true, the newly created transaction will not be relayed to the monero network. (Defaults to false)
 * *get_tx_hex* - boolean; Return the transaction as hex string after sending
+* *get_tx_metadata* - boolean; Return the transaction metadata.
 
 
 Outputs:
 
 * *fee* - Integer value of the fee charged for the txn.
 * *tx_hash* - String for the publically searchable transaction hash
 * *tx_key* - String for the transaction key if get_tx_key is true, otherwise, blank string.
+* *amount_keys* - List of strings for the amount keys.
+* *amount* - Amount transferred for the transaction.
 * *tx_blob* - Transaction as hex string if get_tx_hex is true
-
+* *tx_metadata* - Transaction metadata string if get_tx_metadata is true.
+* *multisig_txset* - Set of signing keys used in a multisig transaction (empty for non-multisig).
```

Comment:
Incorrect, it is a set of transactions in the process of being signed.

Reply:
Ok, will update, don't know why I thought it was the keys.